vmdb/app/models/service_template.rb

```diff
@@ -52,6 +52,10 @@ def request_type
     "clone_to_service"
   end
 
+  def service_class_type
+    nil  # base class
+  end
```

Comment:
Instead of returning `nil` here shouldn't we return Service.  With the naming convention this could be
```ruby
class.name.sub('Template', '')
```
then you do not need to override in the subclass anymore.